contracts/BatchAuction.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.0;
+
+
+contract BatchAuction {
+
+    uint16 public maxAccountNumber = 10000;
+
+    mapping (address => uint16) public publicKeyToAccountMap;
+    mapping (uint16 => address) public accountToPublicKeyMap;
+
+    event AccountOpened(uint16 indexed depositor);
```

Comment:
this event is unused

Reply:
Yes, you caught this just as I was about to comment myself! Should it be used or removed? if I recall correctly, this wont be necessary for the DB.